### ISSUE TITLE
Display error with lots of warnings on the browser console

### ISSUE THREAD

**Issue by eldariont** [vgteam/sequenceTubeMap#65]
Hi,
I have problems displaying alignments of short reads (~30x) on a simple yeast vg graph. Both files can be found here:  http://public.gi.ucsc.edu/~daheller/tubemap/

This is what I did:
I started the docker container (tried both with wolfib/sequencetubemap:vg2018 and wolfib/sequencetubemap:vg-1.6.0-382) and ran both prepare scripts on the files as described in the README. In the frontend interface I chose "Data: custom; xg file: construct.vg.xg; gam index: SRR4074358.subsampled30.mapped.gam.index; Path name: chrI; Start: 70000" and clicked "Go".

After loading, the graph looks really weird. I see nodes and alignments and can scroll horizontally. However, I cannot zoom or scroll vertically and the position numbers aren't visible (see screenshot below). What I get are lots of errors/warnings on the console like "Unexpected value translate(4,NaN) scale(1) while parsing attribute transform."

Any idea of what went wrong or what I am doing wrong?

Cheers
David

![grafik](https://user-images.githubusercontent.com/6477692/47825462-4a076480-dd2f-11e8-8216-83dc9586b7ca.png)

**Comment by adamnovak** [comment]
That looks like quite an old version, since there's no zoom buttons. I think maybe we want to fix #66 first and see if this recurs.

**Comment by wolfib** [comment]
Yes, the docker container is not up to date and I am trying to get rid of needing it by providing an alternative installation path.

But I'm afraid there is something going on in that data which causes a bug or some strange behavior. I just had a look at your files, and they mostly render ok, but at around position 70000 I see this:
<img width="1942" alt="screen shot 2018-11-02 at 19 13 19" src="https://user-images.githubusercontent.com/100017/47933089-d9e61380-ded3-11e8-851e-53ddd35b6439.png">

I need some more time to find out what's going on there.

**Comment by eldariont** [comment]
Thanks for looking into this.
Yes, the docker containers I used were 9 and 12 months old.

About the data: I generated the graph with `vg construct` and a VCF file that among other SVs contains a 30bp insertion at position 71012. Then I aligned short reads against that graph. Your screenshot seems to show the insertion but the read alignments look weird. I'm curious to know where the problem is.

**Comment by wolfib** [comment]
This should be fixed with c4c7b3297ac5b815fdabbf33fe01957783f90524. 

There was a bug with how nodes which contain only reads and no haplotypes were placed.
<img width="1505" alt="screen shot 2018-11-05 at 13 45 21" src="https://user-images.githubusercontent.com/100017/47998908-13e82d00-e101-11e8-9325-397f9216659d.png">
